### PR TITLE
[MS] Add description on dropdown user role

### LINKED ISSUE #5887: Implement MsDropdown subtitle use cases
- [x] Update mock-ups
- [x] Apply mock-ups use cases (workspace user role choices for example)

### PR DESCRIPTION
**Pull request by fabienSvtr**
### What have changed?
The user can understand what the role or profile can do throughout the application.

### Where: 
* Greet new user (profile choice)
* Share workspace (role choice)

### Describe your changes (screenshots)
image
image

## Checklist

Before you submit this pull request, please make sure to:

- [x] Keep changes in the pull request as small as possible
- [x] Ensure the commit history is sanitized
- [x] Give a meaningful title to your PR
- [x] Describe your changes
- [x] Link any related issue in the description
- [x] Link any dependent pull request in the description

closes #5887

### CODE REVIEW

File: client/src/locales/fr-FR.json [hunk 1]
```diff
@@ -656,9 +668,18 @@
         "inviteUser": "Inviter un utilisateur",
         "itemCount": "Aucun utilisateur | Un utilisateur | {count} utilisateurs",
         "profile": {
-            "admin": "Administrateur",
-            "standard": "Standard",
-            "outsider": "Externe"
+            "admin": {
+                "label": "Administrateur",
+                "description": "Can manage the organization and its users."
```
Comment (Max-7) [inline client/src/locales/fr-FR.json:673]: Missing translation.

**Review by Max-7 — CHANGES_REQUESTED** [review]

**Review by NicoTuxx — CHANGES_REQUESTED** [review]

File: client/src/common/translations.ts [hunk 2]
```diff
@@ -1,24 +1,39 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
+import { MsWorkspaceRole } from '@/components/core/ms-types';
```
[the remark this answers is not in the document]
Comment (fabienSvtr) [reply client/src/common/translations.ts:3]: Done it.

File: client/src/common/translations.ts [hunk 3]
```diff
@@ -1,24 +1,44 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
+import { MsOption } from '@/components/core/ms-types';
+import { InvitationStatus, WorkspaceRole } from '@/parsec';
 import { ComposerTranslation } from 'vue-i18n';
-import { WorkspaceRole, InvitationStatus } from '@/parsec';
 
-export function translateWorkspaceRole(t: ComposerTranslation, role: WorkspaceRole | null): string {
+export function translateWorkspaceRole(t: ComposerTranslation, role: WorkspaceRole | null): MsOption {
```
Comment (Max-7) [inline client/src/common/translations.ts:7]: This should not return a `MsOption`. Just define a dummy interface above, something like
```typescript
interface WorkspaceRoleTranslation {
  label: string;
  description?: string;
}
````
and use this as a return value. It doesn't need to be exported, it's never going to be used outside this function, and it's fine.

File: client/src/components/core/ms-types.ts [hunk 4]
```diff
@@ -12,7 +12,7 @@ export enum MsModalResult {
   Confirm = 'confirm',
 }
 
-interface MsOption {
+export interface MsOption {
```
Comment (Max-7) [inline client/src/components/core/ms-types.ts:15]: No export needed.

**Review by Max-7 — CHANGES_REQUESTED** [review]

**Review by NicoTuxx — APPROVED** [review]

**Review by Max-7 — APPROVED** [review]

**Pull request merged**